### PR TITLE
modern_pascal_introduction_Chinese.pdf

### PR DESCRIPTION
**Pull request by robsean**

### CODE REVIEW

**Comment by michaliskambi** [comment]
Wow! First of all, thank you -- that's a huge work, amazing!

My main question is could you remake your translation to be an AsciiDoc file, not PDF file. 

The "source" AsciiDoc file in English is `modern_pascal_introduction.adoc` in the repository. You can open it in any text editor (Emacs, Atom, Notepad++...). 

Our preferred way to translate is to make a new AsciiDoc with translation, so you would make `modern_pascal_introduction_chinese.adoc` (similar to existing `modern_pascal_introduction.adoc` in English and `modern_pascal_introduction_russian.adoc` in Russian). This way you do not have to commit PDF, I only need the `modern_pascal_introduction_chinese.adoc` in AsciiDoc, and from there I can generate any output -- HTML, PDF.

I know it's a large change that I'm asking. The reason we want it like this is that:

- I want to be able to generate various outputs, not only HTML and PDF. I also want the outputs to look "consistent", with a similar style and colors, link to my Patreon etc.

- Others should be able to improve the translation later too, eventually. PDF is hard to edit, while AsciiDoc is easy to further edit.

- Changes in version control (GIT) are nicely viisble for AsciiDoc, they are not visible in PDF nicely.

Please let me know if you have any questions how to do it :)

**Comment by robsean** [comment]
OK， I will remake thetranslation to be an AsciiDoc file

**Comment by michaliskambi** [comment]
Any news about this?

I'd hate to see the huge translation effort you did not being used. So, if you don't plan to update it, I may just put a link to the PDF somewhere.

Still, only having a ready PDF is very non-optimal. It means that the Chinese translation cannot really be updated by anyone, we don't have a HTML version, we cannot even update examples there -- it's all baked in the PDF file, which is very hard to edit. We want to edit an AsciiDoc version instead.

**Comment by michaliskambi** [comment]
I merged the PR, and I made a prominent link to this Chinese translation from our website: see 

- https://castle-engine.io/modern_pascal sidebar, 

- and the dedicated translations list on https://castle-engine.io/modern_pascal_translations .

If you (or anyone else listening on this GitHub issue :) ) have time, I would still welcome the improvement I mentioned above: I'd like to convert this translation into AsciiDoctor (which is easier to edit by everyone, and from it we can easily generate HTML and PDF outputs). You're welcome to see how other translations are organized in https://github.com/michaliskambi/modern-pascal-introduction/ now. Not knowing Chinese, I cannot do this alone, but I can help -- if you have any questions how to convert something to AsciiDoctor just ask :) 

Basically, the idea is to eventually delete the `modern_pascal_introduction_chinese.pdf` from the repository, but create `modern_pascal_introduction_chinese.adoc`. From the adoc file, we can always automatically generate both HTML and PDF outputs.

That said, for now I take what we have, and enjoy it :) Thank you very much for making the translation!

If you'd like to improve anything in the above pages (e.g. how I credit you on https://castle-engine.io/modern_pascal_translations ) naturally go ahead and ping me. You can just edit this file in cge-www repository: https://github.com/castle-engine/cge-www/blob/master/htdocs/doc/modern_pascal_translations.adoc and submit a pull request.
